packages/components/rollup.config.mjs

```diff
@@ -130,12 +130,14 @@ export default {
     {
       dir: "dist",
       entryFileNames: "[name].cjs",
+      chunkFileNames: "[name]-[format].js",
```

Comment:
I am not entirely sure about the reason for this change? It looks like the default behaviour in [rollup](https://rollupjs.org/configuration-options/#output-chunkfilenames) is to have it be a hash